contracts/SpokePool.sol

```diff
@@ -480,41 +532,60 @@ abstract contract SpokePool is SpokePoolInterface, Testable, Lockable, MultiCall
             relayFills[relayHash] += fillAmountPreFees;
             // If relay token is weth then unwrap and send eth.
             if (relayData.destinationToken == address(weth)) {
-                IERC20(relayData.destinationToken).safeTransferFrom(msg.sender, address(this), amountToSend);
+                if (!isSlowRelay)
```

Comment:
Can you add a comment explaining that the source of funds to pay the recipient is either the caller+relayer or this contract depending on whether this is a slow relay

Reply:
yep!